lib/internal/modules/esm/initialize_import_meta.js

```diff
@@ -32,7 +32,7 @@ function createImportMetaResolve(defaultParentURL, loader, allowParentURL) {
         case 'ERR_MODULE_NOT_FOUND':
           ({ url } = error);
           if (url) {
-            return url;
+            return `${url}`;
```

Comment:
Please let's fix this in the error creation in `fetch_module.js` on the 404 path where it assigns the URL not the string. The error type should also be a string to begin with as well.